CMakeLists.txt

```diff
@@ -458,42 +477,29 @@ endfunction()
 
 if(CMAKE_CROSSCOMPILING)
 
-	file(GLOB FW_VARIANTS RELATIVE ${PROJECT_SOURCE_DIR}/Firmware/variants ${PROJECT_SOURCE_DIR}/Firmware/variants/*.h)
-	foreach(THIS_VAR IN LISTS FW_VARIANTS)
-		string(REPLACE ".h" "" TRIMMED_NAME "${THIS_VAR}")
-		message("Variant added: ${TRIMMED_NAME}")
-        string(REPLACE "-E3Dv6full" "" DIR_NAME "${TRIMMED_NAME}")
```

Comment:
this was done so as not to break the existing builds... I'd leave it until we actually make the change, the prefix and postfix are just garbage/noise on screen and add to directory lengths/paths to anannoying degree.

Reply:
nevermind - stupid diff view is terrible sometimes.